tutorials/kubeflow-on-kapsule/index.mdx

```diff
@@ -29,35 +28,37 @@ In this tutorial you will learn how to deploy Kubeflow and its services on [Kube
+2. Click **Kubernetes** on the side menu.
 
 3. The Kubernetes section displays. Click **Create a cluster** to create a new Kubernetes cluster.
 
-  <Lightbox src="scaleway-kubeflow_kapsule_console_new.png" alt="" />
+4. The Kubernetes section displays. Click **Create a cluster** to create a new Kubernetes cluster.
+
+
+    - **Choose your cluster type**: Choose a Kubernetes Kapsule cluster.
+    - **Choose a region**: Choose the region in which you want your cluster to be created. A region refers to the geographical location in which your cluster will be created.
+    - **Choose a Kubernetes Version**: Select the Kubernetes version for the cluster. Choose the latest version from the drop-down list.
+    - **Enter a Name for the Cluster**: Name your cluster or leave the automatically generated name. 
+    - Keep the default values in the `Advanced Options` section.
+    - Click **Next** to configure your pool.
+    - **Choose a zone**: An [Availability Zone (AZ)](/compute/instances/concepts/#availability-zone) refers to the geographical location in which all your pool's nodes will be created.
+    - **Select a node type**: Choose the node type. We will be using `GP1-M` Instances for this tutorial (16 cores, 64 GB RAM).
+    - **Configure node options**:
+      - Enable the autoscale feature.
+      - Enter the minimum and maximum number of nodes available for the pool. Choose a minimum of **1 node** and a maximum of **2 nodes**. 
+
+<Message type="note">
 
-4. The cluster creation page displays. Enter the following information:
 
-    - **Choose your cluster type**: Choose between Scaleway Kubernetes Kapsule or Kubernetes Kosmos.
-    - **Choose a region**: A region refers to the geographical location in which your cluster will be created.
-    - **Choose a Kubernetes Version**: Select the Kubernetes version for the cluster. Choose `Version 1.18.16` from the drop-down list. This version is [compatible](https://www.kubeflow.org/docs/started/k8s/overview//) with Kubeflow.
-    - **Enter a name for the cluster**: Set a name for the cluster, for example `Kubeflow`.
-    Click **Next** to configure your pool.
-    - **Choose a zone**: An Availability Zone (AZ) refers to the geographical location in which all your pool's nodes will be created.
-    - **Select a node type**: We will use `GP1-M` Instances for this tutorial (16 cores, 64 GB RAM)
-    - **Configure node options**: Choose the number of nodes for your default pool
-      - Activate `Autoscale the number of nodes`
-      - Enter the minimum and maximum number of nodes available for the pool, we choose `Minimum 1` and `Maximum 2`
+ In case you choose an under dimensioned configuration for your base cluster, and if you later add GPU nodes to your pool with autoscaling, the automatic scaling down of the GPU nodes might be prevented by Kubeflow system pods being deployed on the GPU nodes (due to the high cpu/memory pressure on the CPU nodes).
```

Comment:
```suggestion
 If you choose an under-dimensioned configuration for your base cluster, and you later add GPU nodes to your pool with autoscaling, you may have problems with automatic downscaling of the GPU nodes. This can happen when  Kubeflow system pods deployed on the GPU nodes prevent downscaling, due to the high CPU/memory pressure on the CPU nodes.
```